conbench/tests/entities/test_commit.py

```diff
@@ -129,28 +129,39 @@ def test_repository_to_name():
 
 
 def test_repository_to_url():
+    """'
+    Currently, repository_to_url() has too many features. We are actively
+    narrowing down the input parameter space.
+    """
     expected = "https://github.com/apache/arrow"
     assert repository_to_url(None) == ""
     assert repository_to_url("") == ""
     assert repository_to_url("blah blah") == "https://github.com/blah blah"
     assert repository_to_url("apache/arrow") == expected
-    assert repository_to_url("https://github.com/apache/Arrow") == expected
+    # Note(JP): this test did confirm that we did some lowering magic, but that
+    # is not wise, also see https://github.com/conbench/conbench/issues/887
+    assert (
+        repository_to_url("https://github.com/apache/Arrow")
+        == "https://github.com/apache/Arrow"
+    )
     assert repository_to_url("https://github.com/apache/arrow") == expected
     assert repository_to_url("git@github.com:apache/arrow") == expected
 
 
-def test_get_github_commit_none():
-    repo = "https://github.com/apache/arrow"
-    sha = "3decc46119d583df56c7c66c77cf2803441c4458"
-    branch = "some_user_or_org:some_branch"
-    pr_number = 123
+# def test_get_github_commit_metadata_none():
+#     repo = "https://github.com/apache/arrow"
+#     sha = "3decc46119d583df56c7c66c77cf2803441c4458"
+#     branch = "some_user_or_org:some_branch"
+#     pr_number = 123
 
-    assert get_github_commit(None, None, None, None) == {}
-    assert get_github_commit("", "", "", "") == {}
-    assert get_github_commit(repo, None, None, None) == {}
-    assert get_github_commit(None, pr_number, None, None) == {}
-    assert get_github_commit(None, None, branch, None) == {}
-    assert get_github_commit(None, None, None, sha) == {}
+#     # What was this supposed to confirm? That indeed all detail needs
+#     # to be provided?
```

Comment:
I think simply that this function wouldn't fail if given various missing inputs. We can delete this test